src/D2L.Bmx/WriteHandler.cs

```diff
@@ -18,6 +21,7 @@ BmxConfig config
 		_awsCreds = awsCreds;
 		_consolePrompter = consolePrompter;
 		_config = config;
+		_parser = new FileIniDataParser();
```

Comment:
pleas pass it through the constructor parameter if only for consistency.

Reply:
Is here where you mean? https://github.com/Brightspace/bmx/blob/main/src/D2L.Bmx/Program.cs#L131

Reply:
yeah

Reply:
Also, I think we can reuse this ini parser in `BmxConfigProvider` and get rid of `Microsoft.Extensions.Configuration.Ini`.
There are some nice things we can do with a proper parser+writer.
Another time though.

Reply:
Thanks for clarification. I've moved it